test/conformance/pathrules/feature.go

```diff
@@ -50,12 +50,38 @@ func InitializeScenario(ctx *godog.ScenarioContext) {
 	})
 }
 
-func anIngressResourceInANewRandomNamespace(arg1 *messages.PickleStepArgument_PickleDocString) error {
-	return godog.ErrPending
+func anIngressResourceInANewRandomNamespace(spec *messages.PickleStepArgument_PickleDocString) error {
+	ns, err := kubernetes.NewNamespace(kubernetes.KubeClient)
```

Comment:
Do we want to repeat the code or call a util function for both of these?
Can we stick to a single way to do things between the different conformance packages?

Reply:
> Can we stick to a single way to do things between the different conformance packages?

Sure, but for that, we need to be consistent in the definition of the features. Not sure we can do that now.
Maybe we can delay this until we have all the features?